markdown/usage/troubleshooting.md

```diff
@@ -84,20 +149,106 @@ Alternatively, you can also add Singularity Bind Paths to your Nextflow call, e.
 If you get a warning like the following:
 
 ```bash
-Project nf-core/eager currently is sticked on revision: dev -- you need to specify explicitly a revision with the option -r to use it
+Project nf-core/<pipeline> currently is sticked on revision: dev -- you need to specify explicitly a revision with the option -r to use it
 ```
 
-This is a nextflow error, with less commonly seen terminology. What this means is that you have multiple versions of nf-core/eager pulled (e.g. 2.0.0, 2.1.0, 2.1.1, dev etc.). When you have multiple versions, you must always specify which one you want to use - there is no 'default'. Therefore, with every `nextflow run nf-core/<PIPELINE>` command you must always indicate which version with `-r`.
+This is a Nextflow error, with less commonly seen terminology. What this means is that you have multiple versions of the pipeline pulled (e.g. 2.0.0, 2.1.0, 2.1.1, dev etc.). When you have multiple versions, you must always specify which one you want to use - there is no 'default'. Therefore, with every `nextflow run nf-core/<PIPELINE>` command you must always indicate which version with `-r`.
```

Comment:
Isn't default actually the default branch on GitHub, so actually `master` for a released pipeline, so the latest released pulled?

Reply:
Ah I see - I meant once you specify a specific version, there is no more 'default' (and you always have to specify which you are pulling).

Reply:
Oh wierd, I don't get that error anymore.

Reply:
I sometimes have this error when there's a new release or the `dev` is updated and I try to run without pulling it first.

But I see what you mean, and I have no idea what should actually happen there.